app/views/events/show.html.haml

```diff
@@ -53,18 +53,20 @@
       %p.text-center
         %i= t('events.thx_to_sponsors')
       %br
-      - if @event.sponsors.any?
+      - if @event.sponsors?
         %ul.no-bullet
-          - @event.sponsors.each do |sponsor|
-            %li
-              .row
-                .large-2.columns
-                  = image_tag(sponsor.avatar.url, class: 'sponsor', alt: sponsor.name)
-                .large-10.columns
-                  =link_to sponsor.name, sponsor.website
-                  %p
-                    = sponsor.description
-
+          - if @event.sponsors?(:gold)
+            %h3.text-center Gold 
+            = render partial: "sponsors", object: @event.gold_sponsors
+          - if @event.sponsors?(:silver)
+            %h3.text-center Silver
+            = render partial: "sponsors", object: @event.silver_sponsors
+          - if @event.sponsors?(:bronze)
+            %h3.text-center Bronze
+            = render partial: "sponsors", object: @event.bronze_sponsors
+          - if @event.sponsors?(:standard)
+            %h3.text-center Standard
```

Comment:
If the event only has standard sponsors, should it still show the title? Question for @KimberleyCook